packages/rum-core/src/performance-monitoring/metrics.js

```diff
@@ -215,6 +314,11 @@ export function captureObserverEntries(list, { capturePaint }) {
     result.spans.push(tbtSpan)
   }
 
+  const fidSpan = createFirstInputDelaySpan(list)
```

Comment:
Lets follow the same semantics as other functions in this file. Please pass the entries here instead of the list. 

```
const fidEntries = list.getEntriesByType(FIRST_INPUT)
const fidSpan = createFirstInputDelaySpan(fidEntries)
```